sorc/ncep_post.fd/PROCESS.f

```diff
@@ -54,79 +54,66 @@ SUBROUTINE PROCESS(kth,kpv,th,pv,iostatusD3D)
 !     START SUBROUTINE PROCESS.
 !
       cfld=0
-      if(me==0) write(*,*) "PROCESS starts"
```

Comment:
@DusanJovic-NOAA I would like to keep the prints in this subroutine for tracking the process status. They are fewer than 15 lines.

Reply:
Reverted in [8d3255a](https://github.com/NOAA-EMC/UPP/pull/705/commits/8d3255a605430769debbfa6e76ecfa6899171f44)